src/js/pages/Records-page.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+const Page = require('./page')
+
+class RecordsPage extends Page {
+  get url () {
+    return '/records'
+  }
+
+clickRecords() {
+  const link = $('=Records')
+  link.click()
+}
+ }
```

Comment:
The spacing looks off here. Run `npm run lint` and it should show you the errors.